idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/tests/test_column.py

```diff
@@ -0,0 +1,297 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2019, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Author: Paul Akula, Anuja Deshpande
+"""
+# Import Python libraries
+import sys
+import os
+import pytest
+
+# Import Pyomo libraries
+from pyomo.environ import ConcreteModel, value, Param, TransformationFactory,\
+    SolverFactory , TerminationCondition, units as pyunits
+
+# Import IDAES Libraries
+from idaes.core import FlowsheetBlock
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+# Access the mea_solvent_system dir from the current dir (tests dir)
+sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
+
+from unit_models.column import PackedColumn, ProcessType
+from properties.vapor_prop import VaporParameterBlock
+from properties.liquid_prop import LiquidParameterBlock
+
+# -----------------------------------------------------------------------------
+solver = SolverFactory('ipopt')
```

Comment:
We normally use the `get_default_solver_ method from `idaes.core.util.initialization` here. That points to IPOPT now, but allows us to easily change later if there is a reason.